modules/views/civicrm.views.inc

```diff
@@ -304,6 +304,9 @@ function civicrm_views_add_date_arguments(&$data, $value) {
  *   Array with the new custom field appended
  */
 function civicrm_views_custom_data_cache(&$data, $entity_type, $group_id, $sub_type) {
+  if (!CRM_Core_Config::isUpgradeMode()) {
```

Comment:
@seamuslee001 so if it doesn't seem to make a difference whether ! or not - then how can we get to the rest of this function if we're not upgrading?